editor/src/clj/editor/gui.clj

```diff
@@ -2723,7 +2730,7 @@
 (g/defnode LayoutsNode
   (inherits outline/OutlineNode)
   (input names g/Str :array)
-  (output names g/Any (gu/passthrough names))
+  (output names GuiResourceNames :cached (g/fnk [names] (into (sorted-set) names)))
```

Comment:
We can collect the names into a set directly in the `LayoutsNode`. This connects to the `layout-names` input `GuiSceneNode`.